### PR TITLE
v0.2.6

### PR DESCRIPTION
**Pull request by sonichi**
xgboost notebook

### CODE REVIEW

File: flaml/searcher/search_thread.py [hunk 1]
```diff
@@ -70,7 +71,7 @@ def _update_speed(self):
         # calculate speed; use 0 for invalid speed temporarily
         if self.obj_best2 > self.obj_best1: 
             self.speed = (self.obj_best2 - self.obj_best1) / (
-                self.cost_total - self.cost_best2)
+                self.cost_total - self.cost_best2 + self.eps)
```
Comment (qingyun-wu) [inline flaml/searcher/search_thread.py:74]: why adding self.eps? (presumably self.cost_total should not equal to self.cost_best2)

Comment (sonichi) [reply flaml/searcher/search_thread.py:74]: test_nested fails in test_tune.py with DivideByZeroException

Comment (qingyun-wu) [reply flaml/searcher/search_thread.py:74]: why does it fail?

Comment (sonichi) [reply flaml/searcher/search_thread.py:74]: self.cost_total = self.cost_best2

Comment (qingyun-wu) [reply flaml/searcher/search_thread.py:74]: So we have self.obj_best2 > self.obj_best1 and  self.cost_total = self.cost_best2 at the same time?

Comment (sonichi) [reply flaml/searcher/search_thread.py:74]: yes

**Review by qingyun-wu — APPROVED** [review]

**Review by qingyun-wu — APPROVED** [review]

**Pull request merged**